Tests/Unit/MSFT_xWindowsFeature.Tests.ps1

```diff
@@ -1,205 +1,215 @@
 # These tests use a mock server module. They will fail on an actual server.
 # All tests that require a credential will be skipped
 
-Initialize-TestEnvironment `
-    -DSCModuleName 'xPSDesiredStateConfiguration' `
-    -DSCResourceName 'MSFT_xWindowsFeature' `
-    -TestType Unit `
-    | Out-Null
+Import-Module -Name (Join-Path -Path (Split-Path $PSScriptRoot -Parent) `
```

Comment:
Fair question. We're trying to switch to using Join-Path instead of '\' for a couple reasons. The main one is if we go cross-platform with these resources, there's often different ways of resolving file paths. For instance Windows vs. Linux.